### PR TITLE
add more descriptive error message for ReadWriteConflictErrors

### PR DESCRIPTION
**Pull request by tanishqkancharla**

### CODE REVIEW

File: src/database/ConcurrencyLog.ts [hunk 1]
```diff
@@ -9,8 +10,14 @@ type WriteItem = { type: "write"; tuple: Tuple; txId: TxId | undefined }
...
 
 export class ReadWriteConflictError extends Error {
-	constructor(txId: string | undefined) {
-		super("ReadWriteConflictError: " + txId)
+	constructor(txId: string | undefined, writeTuple: Tuple, readBounds: Bounds) {
+		const message = outdent`
+      ReadWriteConflictError: ${txId}
+      Write to tuple ${writeTuple}
+      conflicted with a read at the bounds ${readBounds}
+    `
```
Comment (ccorcos) [inline src/database/ConcurrencyLog.ts:18]: Why not `outdent(...)`?

Comment (tanishqkancharla) [reply src/database/ConcurrencyLog.ts:18]: 🤷🏾‍♂️ that was the author's choice of api: https://www.npmjs.com/package/outdent

Comment (tanishqkancharla) [reply src/database/ConcurrencyLog.ts:18]: you would need to do `outdent.string(...)`

**Comment by ccorcos** [comment]
Run `git reset master -- package.json package-lock.json` to clean everything up

**Comment by tanishqkancharla** [comment]
Got it. Since i'm making a PR from my fork, I had to add another remote "upstream" and reset from there.

**Pull request merged**